### ISSUE TITLE
Clarification on what documentation can be included in o3de.org

### ISSUE THREAD

**Issue by chanmosq** [o3de/sig-docs-community#72]
See topic https://github.com/o3de/sig-docs-community/issues/60#issuecomment-1239990077

**Comment by chanmosq** [comment]
We discussed this at [sig-docs-community meeting on 11/08/2022](https://github.com/o3de/sig-docs-community/issues/68).

We are in agreement on the following things in regards to docs for third-party features:
- Docs for third-party (non-core) features of O3DE are not to be maintained in O3DE Docs. Because the state of that feature is not maintained in O3DE's source code, it's documentation also is out-of-scope for O3DE Docs. 
- Although we do not maintain such docs, we can maintain a page in the O3DE Docs that **links** to docs for third-party features. Linked docs should be complete and trustworthy sources. (E.g. Docs written by an O3DF partner who's contributed a third-party Gem and maintains the docs in their website.)

There are still some open questions:
- **What about docs for a third-party feature that was written by a contributor, not the owner of the feature themselves?**

  Such docs are highly appreciated contributions, but due to ambiguity on ownership of maintenance of those docs, we request that they are not part of the official O3DE Docs. Instead, such docs may be a good candidate for a blog post. The contributor would be advised to speak with the community side of sig-docs-community to verify if blogs is an appropriate place for their doc and make this arrangement. 
**Example case**: A document about an artist workflow for creating 3D assets for O3DE in Blender. This document is highly dependent on the state of Blender, which is out of our control and should be kept out of O3DE Docs. However, this may be a good candidate for a blog post. 
_Note: This question is still open because it's not an established practice and depends on O3DE community's thoughts._

- **What do we do with docs about third-party features that are already in O3DE Docs?**
  We don't have an answer for this at this time. This may require a larger discussion with the O3DF, perhaps through TSC. Docs that affect this are currently the AWS Gems and Kythera AI tutorial.

**Comment by chanmosq** [comment]
See notes from meeting: https://github.com/o3de/sig-docs-community/issues/84#issuecomment-1349869774
